src/axolotl/monkeypatch/models/qwen3_moe/modeling.py

```
@@ -0,0 +1,253 @@
+                    if bias_bank is not None:
+                        bias_active = bias_bank
+                        out = out + torch.repeat_interleave(bias_active, repeat_sizes, dim=0).to(out.dtype)
+                    return out
+                except (RuntimeError, NotImplementedError) as exc:
+                    if isinstance(exc, RuntimeError) and "grouped gemm is not supported" not in str(exc).lower():
+                        raise
+
+            outputs: list[torch.Tensor] = []
+            start_idx = 0
+            bias_iter = bias_bank if bias_bank is not None else [None] * weight_bank.size(0)
+            for count, weight, bias in zip(repeat_sizes.tolist(), weight_bank, bias_iter):
```

Ruff (0.14.4)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)